server/verify.go

```diff
@@ -342,7 +342,7 @@ func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTec
 		var snpOpts *VerifySnpOpts
 		tee, ok := attestation.TeeAttestation.(*pb.Attestation_SevSnpAttestation)
 		if !ok {
-			return fmt.Errorf("TEE attestation is %T, expected a SevSnpAttestation", attestation.GetTeeAttestation())
```

Comment:
should this be behind a flag? something like `RequireHardwareAttestation` or `RequireTEEHardwareAttestation`?

Reply:
Yea, a flag makes sense, let me add that!

Reply:
Sorry, just had this thought: should this be the other way around? That way the default requires hw attestation. (e.g., `AllowSoftwareTEEAttestation` or `AllowGCESoftwareTEEAttestation` since it's only for GCE-based sw attestation)

Reply:
(you'll need to add some new tests either way)